### PR TITLE
DEVDOCS-5825: [new] Create store-metafields

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5825]

## What changed?
Added GraphQL and REST store metafields 

## Release notes draft

* 

## Anything else?

ping {names}

[DEVDOCS-5825]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kskomro — APPROVED** [review]
LGTM

File: docs/store-operations/catalog/graphql-admin/store-metafields [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Store metafields
+
+Store metafields allow you to attach a metafield to the store itself. They are suitable for any global information against the store that is not channel specific. Channel metafields are appropriate for any storefront-specific data you need to store in metafields. There are two ways to create a store metafield in Admin GraphQL. You can remove the resource ID or use the store’s Global ID. Examples of each type are below:
```
Comment (bookernath) [inline docs/store-operations/catalog/graphql-admin/store-metafields:3]: @zvuki  do you think we should recommend one way of doing this over another?

**Review by bookernath — APPROVED** [review]
LGTM. I have not tested the requests myself from these docs, but as long as someone has, this is good to go.

File: docs/store-operations/catalog/graphql-admin/store-metafields (hunk 1 above)
Comment (bc-traciporter) [reply docs/store-operations/catalog/graphql-admin/store-metafields:3]: @zvuki what is the preferred way?

**Pull request merged**